[rospy.client][INFO] 2020-12-25 02:32:19,210: init_node, name[/train_2952_1608863539179], pid[2952]
[xmlrpc][INFO] 2020-12-25 02:32:19,211: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-12-25 02:32:19,212: Started XML-RPC server [http://192.168.0.191:45439/]
[rospy.init][INFO] 2020-12-25 02:32:19,213: ROS Slave URI: [http://192.168.0.191:45439/]
[rospy.impl.masterslave][INFO] 2020-12-25 02:32:19,213: _ready: http://192.168.0.191:45439/
[rospy.registration][INFO] 2020-12-25 02:32:19,214: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-12-25 02:32:19,215: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-12-25 02:32:19,313: registered with master
[rospy.rosout][INFO] 2020-12-25 02:32:19,313: initializing /rosout core topic
[rospy.rosout][INFO] 2020-12-25 02:32:19,319: connected to core topic /rosout
[rospy.simtime][INFO] 2020-12-25 02:32:19,320: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-12-25 02:32:19,331: topic[/bev/temporal_bev_image] adding connection to [http://192.168.0.201:39579/], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,334: topic[/odom] adding connection to [http://192.168.0.201:42567/], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,341: topic[/start_goal_points] adding connection to [http://192.168.0.201:43029/], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,347: topic[/is_start_episode] adding connection to [http://192.168.0.201:44045/], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,351: topic[/scan] adding connection to [http://192.168.0.201:42567/], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,355: topic[/scan] adding connection to [http://192.168.0.201:34435/], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,550: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,551: topic[/episode_watcher/is_finish_episode] adding connection to [/episode_manager], count 0
[rospy.internal][INFO] 2020-12-25 02:32:19,554: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.core][INFO] 2020-12-25 02:35:56,653: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-12-25 02:35:56,660: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-12-25 02:35:56,660: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2020-12-25 02:35:56,660: topic[/episode_watcher/is_finish_episode] removing connection to /episode_manager
[rospy.internal][INFO] 2020-12-25 02:35:56,661: topic[/bev/temporal_bev_image] removing connection to http://192.168.0.201:39579/
[rospy.internal][INFO] 2020-12-25 02:35:56,661: topic[/odom] removing connection to http://192.168.0.201:42567/
[rospy.internal][INFO] 2020-12-25 02:35:56,662: topic[/start_goal_points] removing connection to http://192.168.0.201:43029/
[rospy.internal][INFO] 2020-12-25 02:35:56,663: topic[/scan] removing connection to http://192.168.0.201:42567/
[rospy.internal][INFO] 2020-12-25 02:35:56,664: topic[/scan] removing connection to http://192.168.0.201:34435/
[rospy.internal][WARNING] 2020-12-25 02:35:56,665: Unknown error initiating TCP/IP socket to 192.168.0.201:45595 (http://192.168.0.201:39579/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-12-25 02:35:56,666: Unknown error initiating TCP/IP socket to 192.168.0.201:50063 (http://192.168.0.201:34435/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-12-25 02:35:56,666: Unknown error initiating TCP/IP socket to 192.168.0.201:54959 (http://192.168.0.201:42567/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.topics][ERROR] 2020-12-25 02:35:56,667: Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/topics.py", line 326, in close
    c.close()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 846, in close
    self.socket.close()
AttributeError: 'NoneType' object has no attribute 'close'

[rospy.internal][INFO] 2020-12-25 02:35:56,667: topic[/is_start_episode] removing connection to http://192.168.0.201:44045/
[rospy.impl.masterslave][INFO] 2020-12-25 02:35:56,668: signal-2
[rospy.core][INFO] 2020-12-25 02:35:57,152: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,182: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,213: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,244: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,277: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,305: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,336: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-12-25 02:35:57,346: signal_shutdown [atexit]
